packages/fireproof/src/crdt.ts

```diff
@@ -21,7 +21,7 @@ export class CRDT {
     this.blocks = new TransactionBlockstore(this.name, this.clock, this.opts)
     this.clock.blocks = this.blocks
     this.indexBlocks = new IndexBlockstore((this.opts.persistIndexes && this.name) ? this.name + '.idx' : null, this, this.opts)
-    this.ready = Promise.all([this.blocks.ready, this.indexBlocks.ready]).then(() => {})
```

Comment:
here also

Reply:
I made changes to this file and then did shift+control+p and format document. Is it what triggered this space?